Heroes Handbook, Main File

```diff
@@ -1253,7 +1253,7 @@ Quillsplitter, Stonebreaker, Plainstalker, Spiritwalker
 ### Tauren Traits
 Tauren share certain characteristics no matter what tribe they come from.
 
-***Ability Score Increase.*** Your Constitution score increases by 2.
+***Ability Score Increase.*** Your Constitution score increases by 2, and your Strength or Wisdom score increases by 1.
```

Comment:
```suggestion
***Ability Score Increase.*** Your Strength score increases by 2.
```

Reply:
Since both the base racial features for the tauren lean heavily toward Strength, why not have that as their base ASI? Plus, giving both the +2 ASI and the +1 ASI in the base features breaks with the setup every other D&D race builds by.